### PR TITLE
media file

### LINKED ISSUE #1192: Addition of a Media directory
### Description

We need to add a media directory to store profile images when we are getting images from the user.

### What browser(s) are you seeing the problem on?

Not Applicable

### Further details

It'd be of great use for the dev team once we start storing them in  a categorical way which'd make access to images easily.

### PR DESCRIPTION
**Pull request by topguns837**
Closes #1192
  
## Description

I have added the media directory . The code related to the addition of a media directory were already present in setting.py and urls.py . I have commented out a line in .gitignore due to which I was not able to able to add the media folder to the git repo.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,7 +3,7 @@ VirtualEnv
 */.DS_Store
 *.DS_Store
 .DS_Store
-media/
+#media/
```
Comment (gorkemarslan) [inline .gitignore]: Could you delete the commented line?

**Review by gorkemarslan — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (topguns837) [reply .gitignore]: ok

File: .gitignore [hunk 2]
```diff
@@ -3,7 +3,6 @@ VirtualEnv
 */.DS_Store
 *.DS_Store
 .DS_Store
-media/
```
Comment (gorkemarslan) [inline .gitignore:6]: People should not commit the content of the media folder so you should add:
`/project/media/*`

But that will ignore `.gitkeep` file.

So you should also set the `.gitignore` file such that it ignores the content of `media` folder except for `.gitkeep` file. You can look at what is done for `staticfiles` folder in `.gitignore` for this purpose.

**Review by gorkemarslan — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -210,4 +213,8 @@ dmypy.json
 .vscode
 
 # keep staticfiles directory
-!project/staticfiles/.gitkeep
\ No newline at end of file
+!project/staticfiles/.gitkeep
+<<<<<<< HEAD
+!project/media/.gitkeep
+=======
+>>>>>>> 6cbece5ae023814e69a4ef3868ff8e0691e5fcde
```
Comment (gorkemarslan) [inline .gitignore]: Apparently, you somehow include commit numbers in your commit. Could you remove them and make the file cleaner?

**Review by gorkemarslan — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 4]
```diff
@@ -217,4 +217,4 @@ dmypy.json
 <<<<<<< HEAD
 !project/media/.gitkeep
 =======
->>>>>>> 6cbece5ae023814e69a4ef3868ff8e0691e5fcde
+>>>>>>>
```
Comment (brylie) [inline .gitignore]: Remove conflict annotations

**Pull request closed without merging**